### DISCUSSION TITLE
How to jump to end of python if else block?

### DISCUSSION THREAD

**Discussion by hovnatan** [andymass/vim-matchup#286] [Q&A]
I have

    if ...
        .......
    else:
        ..... 
        .....<END>

If my cursor is on `else` is it possible to jump to <END> of the block? Right now, it jumps back to `if` if I use `%` keyboard shortcut.

**Comment by andymass** [comment]
What if you have code like this:

```
if x:
    print('x')
else:
    print('not x')
```

Where will the cursor go after else?

**Reply by hovnatan** [reply]
to the last character of line 4.

**Comment by andymass** [comment]
You can use `]%` to get to the end of the block.
